backend/src/main/java/de/neuefische/backend/dto/ApplicationResponseDTO.java

```diff
@@ -0,0 +1,13 @@
+package de.neuefische.backend.dto;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class ApplicationResponseDTO {
```

Comment:
I would add the Lomobk `@Data` Annotation for generating the equals, hashCode and toString Method